src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+import classnames from 'classnames';
+import { FilterType } from '../../types/FilterStatus';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filterType: string;
+  handleFilterType: (type: string) => void;
+  todos: Todo[];
+  deleteCompleted: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  filterType,
+  handleFilterType,
+  todos,
+  deleteCompleted,
+}) => {
+  const todosLeft = todos.filter(todo => !todo.completed).length;
+  const todosCompleted = todos.filter(todo => todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${todosLeft} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={classnames(
+            'filter__link',
+            { selected: filterType === FilterType.All },
+          )}
+          onClick={() => handleFilterType(FilterType.All)}
+        >
+          All
+        </a>
+
+        <a
+          data-cy="FilterLinkActive"
+          href="#/active"
+          className={classnames(
+            'filter__link',
+            { selected: filterType === FilterType.Active },
+          )}
+          onClick={() => handleFilterType(FilterType.Active)}
+        >
+          Active
+        </a>
+        <a
+          data-cy="FilterLinkCompleted"
+          href="#/completed"
+          className={classnames(
+            'filter__link',
+            { selected: filterType === FilterType.Completed },
+          )}
+          onClick={() => handleFilterType(FilterType.Completed)}
+        >
+          Completed
+        </a>
+      </nav>
+
+      <button
+        data-cy="ClearCompletedButton"
+        type="button"
+        className="todoapp__clear-completed"
+        onClick={deleteCompleted}
+      >
+        {todosCompleted > 0 && 'Clear completed'}
+      </button>
```

Comment:
the empty button on page if I don’t have completed todos 🥲
<img width="699" alt="Screenshot 2022-10-03 at 23 26 46" src="https://user-images.githubusercontent.com/47606379/193675491-69d72472-4515-4b21-8c71-60afc629785c.png">

Reply:
if I don't add empty button my filters gonna jump from left to the right when I finish any of the todos, if I add this empty button it reduces the amount of pixels they move and it slightly fixes the problem

Reply:
can you check how it works on mate demo?

Reply:
how it looks on [mate demo link](https://mate-academy.github.io/react_todo-app-with-api/):
<img width="737" alt="Screenshot 2022-10-05 at 01 01 35" src="https://user-images.githubusercontent.com/47606379/193938341-be77de77-5145-4193-9c93-f6d91bbf4ea2.png">